### PR TITLE
Version 1.0.0

### PR DESCRIPTION
**Pull request by ihorserba**

### CODE REVIEW

File: sdks/js/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "zello-channels-js-sdk",
   "copyright": "Copyright © 2021 Zello Inc.",
```
Comment (vocoded) [inline sdks/js/package.json:3]: let's update the copyright as well

File: sdks/js/package.json [hunk 2]
```diff
@@ -36,7 +36,7 @@
     "raw-loader": "^0.5.1",
     "rename-output-webpack-plugin": "^1.0.1",
     "sass-loader": "^7.0.1",
-    "scriptjs": "^2.5.8",
+    "scriptjs": "^2.5.9",
```
Comment (vocoded) [inline sdks/js/package.json:39]: are there any other recommended dependency updates we should catch here?

File: sdks/README.md [hunk 3]
```diff
@@ -297,6 +297,7 @@ In iOS, the events interface is `ZCCSessionDelegate`.
 |`Session.incoming_voice_will_start`|This event is fired when an incoming voice stream is about to start. `IncomingMessage` object is returned
 |`Session.incoming_voice_did_start`|Incoming voice message did start (first packet received)
 |`Session.incoming_voice_did_stop`|An incoming voice stream has finished playing.
+|`Session.outgoing_voice_did_stop`|An outgoing voice stream has been stopped by the server
```
Comment (vocoded) [inline sdks/README.md:300]: let's add the other one (fail_start)

**Review by vocoded — COMMENTED** [review]
Final minor requests, then we're good

File: sdks/js/package.json (hunk 2 above)
Comment (ihorserba) [reply sdks/js/package.json:39]: Not easily, but I did some progress: https://github.com/zelloptt/zello-channel-api/pull/187

File: sdks/js/package.json (hunk 1 above)
Comment (ihorserba) [reply sdks/js/package.json]: Updated

File: sdks/README.md (hunk 3 above)
Comment (ihorserba) [reply sdks/README.md:301]: Added

File: sdks/js/src/classes/player.js [hunk 4]
```diff
@@ -13,5 +13,5 @@
  * @example
 player.feed(audioData);
  */
-const PCMPlayer = require('./../vendor/pcm-player/pcm-player');
+const PCMPlayer = require('./../vendor/pcm-player/pcm-player.min');
 module.exports = PCMPlayer;
```
Comment (adambailey-) [inline sdks/js/src/classes/player.js:17]: add newline

Comment (ihorserba) [reply sdks/js/src/classes/player.js]: Added

**Comment by ihorserba** [comment]
Once https://github.com/zelloptt/opus-recorder/pull/7 is re-approved (there were some changes since the last approve) - we have to update the `opus-recorder` submodule

**Comment by ihorserba** [comment]
Last fixes for recorder: https://github.com/zelloptt/opus-recorder/pull/9

**Review by adambailey- — COMMENTED** [review]
Is [80e753f52562eb454910.worker.js](https://github.com/zelloptt/zello-channel-api/pull/184/files#diff-5d69eefe41f40f06ba17bfbc9fd72ac15579dfeed62941db1be031e4a6ec4f31) a desired file we actually want to commit? It seems to me like these files are maybe incidental from our build process, and that we don't actually want them?

**Comment by ihorserba** [comment]
Another small documentation fix
https://github.com/zelloptt/opus-to-pcm/pull/2

**Comment by adambailey-** [comment]
> It's requested as blob before sending and before receiving audio data.

What's the significance behind the name of this?

**Comment by vocoded** [comment]
@adambailey- Nothing we are setting, this is output from emscripten

**Review by adambailey- — APPROVED** [review]

**Comment by ihorserba** [comment]
> > It's requested as blob before sending and before receiving audio data.
> 
> What's the significance behind the name of this?

I'm not sure.
The filename is generated by webpack.

**Comment by ihorserba** [comment]
Back-merged master
Merged the [auth token version support](https://github.com/zelloptt/zello-wss/pull/120)
Rebuilt SDK

File: sdks/README.md [hunk 5]
```diff
@@ -295,8 +295,10 @@ In iOS, the events interface is `ZCCSessionDelegate`.
 |`Session.error`|Error happened
 |`Session.status`|Session received channel status update
 |`Session.incoming_voice_will_start`|This event is fired when an incoming voice stream is about to start. `IncomingMessage` object is returned
-|`Session.incoming_voice_did_start`|Incoming voice message did start (first packet received)
+|`Session.incoming_voice_did_start`|Incoming voice message did start (first packet received and the playback has been initialized correctly)
+|`Session.incoming_voice_fail_start`|This event is called if the audio data was received, but the playback has been failed to start
```
Comment (vocoded) [inline sdks/README.md:299]:
```suggestion
|`Session.incoming_voice_fail_start`|This event is called if the audio data was received, but playback failed to start
```

File: sdks/js/src/classes/sdk.js [hunk 6]
```diff
@@ -84,6 +86,14 @@ ZCC.Sdk.init({
       widget: false
     }, options);
 
+    let scriptsNames = [
```
Comment (vocoded) [inline sdks/js/src/classes/sdk.js:89]: minor, reads better as `scriptNames` (no need for both to be plural)

**Review by vocoded — APPROVED** [review]
minor. are you still waiting on anything to merge?

File: sdks/js/src/classes/sdk.js (hunk 6 above)
Comment (ihorserba) [reply sdks/js/src/classes/sdk.js]: Changed

File: sdks/README.md (hunk 5 above)
Comment (ihorserba) [reply sdks/README.md]: Changed

**Comment by ihorserba** [comment]
Back-merged master, changed according to the latest minor comments; rebuilt SDK

**Comment by ihorserba** [comment]
> minor. are you still waiting on anything to merge?

I think, now we're good to merge

**Comment by ihorserba** [comment]
> minor. are you still waiting on anything to merge?

Oh, I remember the comment [here](https://github.com/zelloptt/zello-channel-api/pull/193) - Maybe add a brief description about possible `tokenVersion` values? Or it's supposed not to be used by SDK users yet?

**Comment by vocoded** [comment]
It can be used, the only supported value is `v1` to indicate the current token format. if omitted, this will be the assumed version.

**Pull request merged**